pkg/ratelimit/limiter.go

```diff
@@ -30,7 +49,10 @@ type Limiter struct {
 
 // NewLimiter returns a global limiter which can be updated in the later.
 func NewLimiter() *Limiter {
-	return &Limiter{labelAllowList: make(map[string]struct{})}
+	return &Limiter{
+		labelAllowList: make(map[string]struct{}),
```

Comment:
Should we regard it as a part of config?

Reply:
I don't think we need to do that, because putting it into configuration would lose the sense of constraint for some API.